Lanchonete/Lanchonete/ Features/Customization/Presentation/CustomizationViewPresenter.swift

```diff
@@ -40,17 +44,27 @@ public class CustomizationViewPresenter {
     //MARK: - Cell Configuration
     func configureIngredientView(_ view: IngredientView, forIndex index: Int){
         let ingredient = ingredients[index]
-        view.display(price: allIngredientManager.getIngredientPrice(ingredient: ingredient))
-        view.display(ingredient: allIngredientManager.getIngredientName(ingredient: ingredient))
-        configureQuantity(view: view, ingredient: ingredient)
+        let quantity = ingredientManager.getQuantity(for: ingredient)
+        
+        view.populate(ingredient: ingredient.name, price: ingredient.price, quantity: quantity, index: index)
     }
     
     func configureQuantity(view: IngredientView,ingredient: Ingredient) {
```

Comment:
If this function doesn't have any implementation, would be nice to remove it from the code base.

Reply:
yes, that's true, I removed it now.